packages/www/components/Plans/index.tsx

```diff
@@ -294,7 +338,25 @@ const Plans = ({ dashboard = false, stripeProductId }: PlanProps) => {
                 displayCheck={false}
                 title={
                   <Flex css={{ justifyContent: "center", width: "100%" }}>
-                    <span>10,000 minutes</span>
+                    <span>5,000 minutes / month</span>
+                  </Flex>
+                }
+                css={{ borderBottom: 0 }}
+              />
+              <Item
+                displayCheck={false}
+                title={
+                  <Flex css={{ justifyContent: "center", width: "100%" }}>
+                    <span>Up to 30</span>
```

Comment:
Missing unit & time range?

Reply:
What are you referring to? This is the row `Concurrent viewers` which this plan (`Sandbox`) has `Up to 30`
Again, exactly the same as the new livepeer.studio pricing page